README.md

```diff
@@ -333,15 +333,15 @@ The files IDE, ITE, IME show the individual effects for the respective individua
 
 RealRate's AI software _Causing_ is a winner of PyTorch AI Hackathon.
 
-<img src="images_readme/RealRate_AI_Software_Winner.png">
+<img src="https://github.com/realrate/Causing/raw/develop/images_readme/RealRate_AI_Software_Winner.png">
```

Comment:
PyPI in itself is not able to get images from relative path so, I have provided a raw GitHub link to the images.

Reply:
This can get confusing when changing the images, since you will always see the images published on the develop branch. But I don't see a way to cleanly handle this, except for not having images in the README (or accepting broken images on PyPI).